web/crux/jest.config.ts

```diff
@@ -0,0 +1,16 @@
+import type { Config } from '@jest/types';
+
+const config: Config.InitialOptions = {
+  moduleFileExtensions: ['js', 'json', 'ts'],
```

Comment:
We need the `'js', 'json',` version?

Reply:
not really, this was the example config. i remove them